bot/seasons/evergreen/issues.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Issues(commands.Cog):
+    """Cog that allows users to retrieve issues from GitHub."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=("issues",))
+    async def issue(self, ctx, number: int, repository: str = "seasonalbot", user: str = "python-discord"):
+        """Command to retrieve issues from a GitHub repository."""
+        url = f"https://api.github.com/repos/{user}/{repository}/issues/{str(number)}"
```

Comment:
String conversion is redundant in this case.

```suggestion
        url = f"https://api.github.com/repos/{user}/{repository}/issues/{number}"
```